src/services/api/UserApiService.ts

```diff
@@ -29,6 +29,29 @@ class UserApiService {
     }
   }
 
+  async getAssignableResponsibles(): Promise<OperationStatus<UserDto[]>> {
```

Comment:
This method can be made more generic. It should have roles as parameters and call the API based on the provided arguments from the context it is called.